README.md

```diff
@@ -236,479 +236,900 @@ The base class sets defaults used by other defined types, such as `tomcat::insta
+
+Valid options: a string.
+
+Default value: the `name` passed in your define.
 
 #####`jvm_route`
 
-Specifies an identifier to enable session affinity in load balancing. Maps to the [jvmRoute XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes). Valid options: string. Default: undef.
+Specifies an identifier to enable session affinity in load balancing. Maps to the [jvmRoute XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes).
+
+Valid options: string.
+
+Default value: `undef`.
 
 #####`jvm_route_ensure`
 
-Specifies whether the [jvmRoute XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes) should exist in the configuration file. Valid options: 'true', 'false', 'present', and 'absent'. Default: 'present'.
+Specifies whether the [jvmRoute XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes) should exist in the configuration file.
+
+Valid options: `true`, `false`, `present`, and `absent`.
+
+Default value: `present`.
 
 #####`parent_service`
 
-Specifies which Service element the Engine should nest under. Valid options: a string containing the name attribute of the Service. Default: 'Catalina'.
+Specifies which Service element the Engine should nest under.
+
+Valid options: a string containing the name attribute of the Service.
+
+Default value: `Catalina`.
 
 #####`server_config`
 
-Specifies a server.xml file to manage. Valid options: a string containing an absolute path. Default: '${catalina_base}/config/server.xml'.
+Specifies a server.xml file to manage.
+
+Valid options: a string containing an absolute path.
+
+Default value: `${catalina_base}/config/server.xml`.
 
 #####`start_stop_threads`
 
-Sets how many threads the Engine should use to start child Host elements in parallel. Maps to the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes). Valid options: a string. Default: undef.
+Sets how many threads the Engine should use to start child Host elements in parallel. Maps to the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes).
+
+Valid options: a string.
+
+Default value: `undef`.
 
 #####`start_stop_threads_ensure`
 
-Specifies whether the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes) should exist in the configuration file. Valid options: 'true', 'false', 'present', and 'absent'. Default: 'present'.
+Specifies whether the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes) should exist in the configuration file.
+
+Valid options: `true`, `false`, `present`, and `absent`.
```

Comment:
present and absent strings, single quote; I'm not going to tag future instances though :-)